pkg/process/executor.go

```diff
@@ -2,111 +2,111 @@ package process
 
 import (
 	"context"
+	"encoding/hex"
 	"fmt"
 
 	capnp "capnproto.org/go/capnp/v3"
-	"github.com/lthibault/log"
 	"github.com/tetratelabs/wazero"
-	"github.com/tetratelabs/wazero/imports/wasi_snapshot_preview1"
+	"lukechampine.com/blake3"
 
+	wasm "github.com/tetratelabs/wazero/api"
 	api "github.com/wetware/ww/internal/api/process"
 )
 
-const start = "_start" // start function called in every WASM module
+// ByteCode is a representation of arbitrary executable data.
+type ByteCode []byte
```

Comment:
I originally named it "binary" because it's the term Wazero uses, but I agree bytecode is more accurate and I am all for it!